internal/config/config.go

```diff
@@ -93,6 +97,9 @@ func LoadConfig(filePath string, fileName string) (*Config, error) {
 	v.SetDefault("nodeIP", "node")
 	v.SetDefault("httpServerPort", 0)
 
+	v.SetDefault("ksm.discovery.retry", 7*time.Second)
```

Comment:
I do not like that much namings, it is not clear that timeout it is the "totalTimeout" and that retry is the time slept between one try and the other

Reply:
whats about "backoffTime"?

Reply:
LGTM, [wikipedia](https://en.wikipedia.org/wiki/Exponential_backoff) uses "delay" to refer to this so I've followed that and used `BackoffDelay`. How does it look to you?

Reply:
Note to self, change the values in the `mapstructure` tag as they are still the old names.